crates/biome_js_analyze/src/assist/source/use_sorted_interface_members.rs

```diff
@@ -0,0 +1,200 @@
+use biome_analyze::{
+    Ast, FixKind, Rule, RuleAction, RuleDiagnostic, RuleSource, context::RuleContext,
+    declare_source_rule,
+};
+
+use biome_console::markup;
+use biome_deserialize::TextRange;
+use biome_diagnostics::{Applicability, category};
+use biome_js_factory::make;
+use biome_js_syntax::{
+    AnyJsObjectMemberName, AnyTsTypeMember, TsInterfaceDeclaration, TsTypeMemberList,
+};
+
+use crate::JsRuleAction;
+use biome_rowan::{AstNode, AstNodeList, BatchMutationExt, SyntaxResult};
+use biome_rule_options::use_sorted_keys::{SortOrder, UseSortedKeysOptions};
+use biome_string_case::comparable_token::ComparableToken;
+use std::borrow::Cow;
+
+trait TsTypeMemberName {
+    fn name(&self) -> SyntaxResult<Option<AnyJsObjectMemberName>>;
```

Comment:
It doesn't make much sense creating a trait that's implemented only for one struct. Let's use a simple function